util/dvsim/FormalCfg.py

```diff
@@ -12,18 +12,18 @@
 from utils import VERBOSE, subst_wildcards
 
 
-class FpvCfg(OneShotCfg):
-    """Derivative class for FPV purposes.
+class FormalCfg(OneShotCfg):
+    """Derivative class for Formal purposes.
```

Comment:
Nit: Stray capital? Maybe "for running formal tools" or something?

Reply:
Yup! Thanks for pointing that out Rupert, updated!